src/Orc.Controls/Orc.Controls.Shared/Controls/DateTimePicker/Views/DateTimePickerControl.xaml

```diff
@@ -4,12 +4,17 @@
                    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
                    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
                    xmlns:controls="clr-namespace:Orc.Controls"
+                   xmlns:converters="clr-namespace:Orc.Controls.Converters"
                    xmlns:catel="http://catel.codeplex.com"
                    mc:Ignorable="d" x:Name="CurrentControl" d:DesignHeight="22" d:DesignWidth="190">
 
+    <catel:UserControl.Resources>
+        <converters:Hour24ToHour12Converter x:Key="_hourConverter" />
```

Comment:
It's needed in code behind. I get it via TryFindResource() method, then depending on Format this converter is enabled or not.